packages/cli/src/databases/repositories/workflow.repository.ts

```diff
@@ -13,16 +13,38 @@ import type {
 import type { ListQuery } from '@/requests';
 import { isStringArray } from '@/utils';
 
+import { FolderRepository } from './folder.repository';
+import type { Folder } from '../entities/folder';
 import { TagEntity } from '../entities/tag-entity';
 import { WebhookEntity } from '../entities/webhook-entity';
 import { WorkflowEntity } from '../entities/workflow-entity';
 import { WorkflowTagMapping } from '../entities/workflow-tag-mapping';
 
+type ResourceType = 'folder' | 'workflow';
+
+type WorkflowAndFolderUnion = {
+	id: string;
+	name: string;
+	name_lower?: string;
```

Comment:
This is temporary to be able to sort the name lowercase with the union and then it's excluded from the response.